etl_manager/meta.py

```diff
@@ -246,9 +247,20 @@ def generate_glue_columns(self, exclude_columns=[]):
                 new_c = {}
                 new_c["Name"] = c["name"]
                 new_c["Comment"] = c["description"]
-                new_c["Type"] = _agnostic_to_glue_spark_dict[
-                    trim_complex_type(c["type"])
-                ]["glue"]
+
+                b1 = c["type"].startswith("array")
+                b2 = c["type"].startswith("struct")
```

Comment:
Should maybe lower them in case the user specified an `ARRAY` or `STRUCT`? Not sure this is necessary as I can't remember if the type check will throw an error with upper case characters

Reply:
👍